orcestra_book/_config.yml

```diff
@@ -30,11 +30,15 @@ sphinx:
   local_extensions:
     apastyle: _ext/
     bracket_citation_style: _ext/
+  extensions:
+#    - sphinx_subfigures
+    - sphinx_panels
```

Comment:
The change doesn't seem to use the `sphinx_panels` extension. Was this an accidental change?

Reply:
During my previous tryouts, I tried `sphinx_panels`. But since I am using cards now, this extension is not needed anymore. I probably forgot to remove this here.

Reply:
Thanks for fixing it :+1: